metatron-app/metatron-discovery
# fn-fix_code_table_error

### Description

2992
if anything is modified in codetable enable reset button and save button otherwise disable
save data without confirm modal

2990
fix delete icon overlap modified date

**Related Issue** : 

METATRON-2992
METATRON-2990

### How Has This Been Tested?

2992
1.Management -> explore data -> code table -> click item
2. change code table data and check buttons are working properly

2990
1. Management -> explore data -> code table 
2. check delete icon is overlap the list when hover above list
#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
